src/core/renderer.js

```diff
@@ -5,6 +5,8 @@ import { Asc, Desc } from './style/functions';
 
 const HISTOGRAM_BUCKETS = 1000;
 
+const initialTimestamp = Date.now();
```

Comment:
why here and not in the constructor?

🌈  uppercase and snake case for global constants

Reply:
why here and not in the constructor?

Because this is just needed for numeric stability, no need to have a per-instance constant